clarity/src/vm/database/clarity_db.rs

```diff
@@ -855,6 +855,31 @@ impl<'a> ClarityDatabase<'a> {
         Ok(())
     }
 
+    fn tenure_height_key() -> &'static str {
+        "_stx-data::tenure_height"
```

Comment:
Can you make sure SIP-021 reflects this new MARF key?  We've done so for the other MARF keys in the Stacks chainstate MARF.